respeaker_ros/scripts/respeaker_node.py

```diff
@@ -115,7 +115,18 @@ def __init__(self):
         if not self.dev:
             raise RuntimeError("Failed to find Respeaker device")
         rospy.loginfo("Initializing Respeaker device")
-        self.dev.reset()
+        try:
+            self.dev.reset()
+        except usb.core.USBError:
+            rospy.logerr(
+                "You have to give the right permission on respeaker device")
+            rospy.logerr(
+                "Please run the command as followings to register udev rules")
+            rospy.logerr(
+                "\n" + "$ roscd respeaker_ros"
+                + "\n" + "$ sudo cp -f $(rospack find respeaker_ros)/config/60-respeaker.rules /etc/udev/rules.d/60-respeaker.rules"
+                + "\n" + "$ sudo systemctl restart udev") # NOQA
+            raise
```

Comment:
you can also print link to https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/master/respeaker_ros/README.md for further information.

https://github.com/ros-drivers/hokuyo_node/blob/c25283fe48ad8eabf6a1b72241ee40cbe0a2cf72/src/hokuyo.cpp#L46-L51

Reply:
I add link in [jsk-ros-pkg/jsk_3rdparty@`dab72eb` (#351)](https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/351/commits/dab72eb3d3cca1658430bdc0d8f888b8186ae268)